src/Status.php

```diff
@@ -19,6 +19,9 @@ public function __construct(string $status)
         $this->status = $status;
     }
 
+    /**
+     * @return string
```

Comment:
Can you add a description to this please.

Reply:
Sure